easybuild/easyconfigs/g/Geant4/Geant4-10.7.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Geant4'
+version = '10.7.1'
+
+homepage = 'https://geant4.cern.ch/'
+description = """Geant4 is a toolkit for the simulation of the passage of particles through matter.
+ Its areas of application include high energy, nuclear and accelerator physics, 
+ as well as studies in medical and space science."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+github_account = 'Geant4'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCELOWER_TAR_GZ}]
+checksums = ['a22800227713b46ba0b71bc674f6c937e5cc0cf17a6becf4a10d5ef90693a60b']
+
+builddependencies = [('CMake', '3.22.1')]
+dependencies = [
+    ('expat', '2.4.1'),
+    # recommended CLHEP version, see https://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.6.html
```

Comment:
fix URL, should be https://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.7.html